content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -9,44 +9,99 @@ weight: 210
 
-* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+* {{< include "task-tutorial-prereqs.md" >}}
 
-* etcd v3.0 or later is required
+* This task assumes that you are running the Kubernetes API server as a
+  {{< glossary_tooltip text="static pod" term_id="static-pod" >}} on each control
+  plane node.
+
+* Your cluster's control plane **must** use etcd v3.x (major version 3, any minor version)
 
 * To encrypt a custom resource, your cluster must be running Kubernetes v1.26 or newer.
 
 
 <!-- steps -->
 
-## Configuration and determining whether encryption at rest is already enabled
+## Determine whether encryption at rest is already enabled {#determining-whether-encryption-at-rest-is-already-enabled}
+
+By default, the `identity` provider used by the API server to protect confidential data
+within etcd.
+**The default `identity` provider does not provide any confidentiality protection.**
 
 The `kube-apiserver` process accepts an argument `--encryption-provider-config`
-that controls how API data is encrypted in etcd.
-The configuration is provided as an API named
-[`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/). `--encryption-provider-config-automatic-reload` boolean argument determines if the file set by `--encryption-provider-config` should be automatically reloaded if the disk contents change. This enables key rotation without API server restarts. An example configuration is provided below.
+that specifies a path to a configuration file. The contents of that file, if you specify one,
+control how Kubernetes API data is encrypted in etcd.
+
+The format of that configuration file is YAML, representing a configuration API kind named
+[`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/).
+You can see an example configuration
+in [Encryption at rest configuration](#understanding-the-encryption-at-rest-configuration).
+
+Check whether the `kube-apiserver` process is running with the `--encryption-provider-config`
+command line argument set to `true`. If it is not, you do not have encryption at rest enabled.
+
+If `--encryption-provider-config` is set, check which resources (such as `secrets`) are
+configured for encryption, and what provider is used.
+Make sure that the preferred provider for that resource type is **not** `identity`; you
+only set `identity` (_no encryption_) as default when you want to disable encryption at
+rest.
+Then the first-listed provider for a resource is something **other** than `identity`,
+and any new information written to resources of that type will be encrypted as configured.
+
+If you are not sure about the progress of any previous migration to encrypt data at rest,
+you must overwrite each resource with a new updated copy, in order to trigger encryption.
+See [Encrypting your data](#encrypting-your-data) for details.
+
+## Encryption-at-rest configuration {#understanding-the-encryption-at-rest-configuration}
```

Comment:
This section has useful information, but where does the content belong?

Reply:
Perhaps that could also happen in a follow-up PR? When I've incorporated wider feedback into PRs, I've found that they then end up at a size that puts people off reviewing them.